docs(design): add design docs from E

timmywil opened the pull request.

timmywil committed:
docs(design): add design docs from E

timmywil commented:
@space-pope I could use an extra set of eyes on these.

space-pope commented on content/docs/Design/for-existing-apps.md:
```diff
@@ -0,0 +1,77 @@
+>
+> **Voice.** This will include audio output. We’ll fill in this row later.
+>
+> **Screen\*\***.\*\* This will include visual output.
```
The trailing asterisks here look a bit strange; I'm not sure what their intent is.

timmywil replied:
```suggestion
> **Screen.** This will include visual output.
```

space-pope commented on content/docs/Design/for-existing-apps.md:
```diff
@@ -0,0 +1,77 @@
+>
+> **Screen\*\***.\*\* This will include visual output.
+>
+> **Support.** This will anything not included in the top three rows (back-end, front-end, marketing, etc.). Keep these broad.
```
```suggestion
> **Support.** This will include anything not included in the top three rows (back-end, front-end, marketing, etc.). Keep these broad.
```

space-pope commented on content/docs/Design/further-learning.md:
```diff
@@ -0,0 +1,36 @@
+draft: false
+---
+
+[Conversational Design](https://www.amazon.com/Conversational-Design-Erika-Hall/dp/1937557677/ref=sr_1_1?keywords=conversational+design&qid=1581373286&sr=8-1)
```
```suggestion
[Conversational Design](https://www.amazon.com/Conversational-Design-Erika-Hall/dp/1937557677)
```

space-pope commented on content/docs/Design/further-learning.md:
```diff
@@ -0,0 +1,36 @@
+[Designing the Future, with Voice](https://vimeo.com/288109032)
+Karen Kaushansky
+
+[Designing Voice User Interfaces: Principles of Conversational Experiences](https://www.amazon.com/Designing-Voice-User-Interfaces-Conversational/dp/1491955414/ref=as_li_ss_tl?ie=UTF8&qid=1515808361&sr=8-1&keywords=designing+voice+user+interfaces&linkCode=ll1&tag=uxma08e-20&linkId=43b7936128cfffd415048e1d95efade3)
```
```suggestion
[Designing Voice User Interfaces: Principles of Conversational Experiences](https://www.amazon.com/Designing-Voice-User-Interfaces-Conversational/dp/1491955414)
```

space-pope commented on content/docs/Design/get-to-know-your-audience.md:
```diff
@@ -0,0 +1,53 @@
+> **Are your users iOS or Android?**
+> This will influence which platform you should/will focus on.
+>
+> **How familiar are your users with voice assistants? How often do they use them? If not, why?**
```
This isn't really an issue with the markdown migration, but "if not, why" doesn't really read well in context. How does "If they don't use voice assistants, why not?" sound, @roppem9?

roppem9 replied:
Works for me

space-pope commented on content/docs/Design/get-to-know-your-audience.md:
```diff
@@ -0,0 +1,53 @@
+
+Take survey results with a grain of salt. Use them as a starting point for further exploration rather than definitive results. You’ll also be able to use this as a recruiting tool when looking for interview and [test candidates](/docs/Design/test-with-real-people). If you want to recruit candidates for [in-person user testing](/docs/Design/test-with-real-people) later on, make sure you ask for lo ...
+
+# **Talk to** **the people** **you’re trying to target**
```
```suggestion
# **Talk to the people you’re trying to target**
```

space-pope commented on content/docs/Design/get-to-know-your-audience.md:
```diff
@@ -0,0 +1,53 @@
+
+Before the interview, create a document or **one-sheet** for each participant. Include the date and the name of the participant in the document’s title. For shared documentation, we use [Google Docs](https://www.google.com/docs/about/). Review survey responses with your team. Write down relevant information under demographics. Here’s a [template](https://docs.google.com/document/d/15YtXuLhlOKrNa6 ...
+
+You’ll be in charge of the discussion. Come prepared with questions, but feel free to improvise. Questions will come to mind as your conversation progresses. We recommend using a script. Here’s an [example](https://docs.google.com/document/d/1KdaeVRv1nlvMvZTMyglkvuv8EG8z6Lcv8Ca9-oLUkjY/edit?usp=sharing) we adopted from Steve Krug’s [Rocket Surgery Made Easy](https://www.amazon.com/Rocket-Surgery- ...
```
```suggestion
You’ll be in charge of the discussion. Come prepared with questions, but feel free to improvise. Questions will come to mind as your conversation progresses. We recommend using a script. Here’s an [example](https://docs.google.com/document/d/1KdaeVRv1nlvMvZTMyglkvuv8EG8z6Lcv8Ca9-oLUkjY/edit?usp=sharing) we adapted from Steve Krug’s [Rocket Surgery Made Easy](https://www.amazon.com/Rocket-Surgery-Made-Easy-Yourself/dp/0321657292). Write down questions specific to a candidate on their one-sheet and make note in your script of when to ask them.
```

space-pope commented on content/docs/Design/get-to-know-your-audience.md:
```diff
@@ -0,0 +1,53 @@
+
+Every project presents a new set of challenges. Need someone to facilitate this process? We can help.
+
+[Email us for more details →](mailto: hello@spokestack.io)
```
The mailto link doesn't look like it's rendering well in github; hopefully it will on the site.

space-pope commented on content/docs/Design/getting-started.md:
```diff
@@ -0,0 +1,34 @@
+
+1. **More control.** A multi-modal approach lets users choose how they interact. Users are able to choose how they input and receive output. This could mean voice-only makes sense in some situations. Gesture-only or some combination of both could make sense in other situations.
+
+2) **Higher success rate**. Multi-modal experiences deliver on a limited domain or set of domains. For example, you're unlikely to find someone asking to have a pizza delivered to their house on a fitness app.
```
```suggestion
2. **Higher success rate**. Multi-modal experiences deliver on a limited domain or set of domains. For example, you're unlikely to find someone asking to have a pizza delivered to their house on a fitness app.
```

space-pope commented on content/docs/Design/map-out-integration.md:
```diff
@@ -0,0 +1,58 @@
+draft: false
+---
+
+Before [writing a script](/docs/Design/script-storyboard-responses), consider what you want users to ask and how you will respond. This is why it’s important to [know your audience](/docs/Design/get-to-know-your-audience). Focus on specific tasks that users can fulfill using voice. For example, buying a movie ticket. Actions the user wishes to perform are called **intents**. Buying a movie ticket ...
```
These links could be relative instead of absolute, but I guess that's not a big deal.

roppem9 replied:
what do you mean?

space-pope replied:
Sorry; that was more for Timmy; wasn't sure if these had been automatically migrated from Dropbox or what—when you're doing links to other docs on the site, if the thing you're linking to is in the same directory as the one you're writing (`docs/Design`, in this case), you can leave off the directory in the link. 

So the ones here would be just `script-storyboard-responses` and `get-to-know-your-audience`. Like I said, though, they point to the same place, so it doesn't really matter.

timmywil replied:
I just missed these.

timmywil replied:
Or, wait, I guess I did change them from absolute with domain to absolute without domain.

timmywil replied:
I'll leave these for now.

space-pope commented on content/docs/Design/map-out-integration.md:
```diff
@@ -0,0 +1,58 @@
+
+Imagine you’re working on a fitness tracker. For the purposes of this exercise, let’s call it MyRunBuddy. Here’s a list of some intents you might include:
+
+![](https://paper-attachments.dropbox.com/s_C4D40CF4264B091F315BA630BFCAF3A9024DF8E8F6C6ED7DCF4188B1D6A5EE61_1580942864791_MyRunBuddy+-+intents.png)
```
This is just the first time I noticed, but should we be hosting these images in the repo?

timmywil replied:
I could be convinced, but it's more work.

space-pope commented on content/docs/Design/map-out-integration.md:
```diff
@@ -0,0 +1,58 @@
+
+An [experience map](https://www.nngroup.com/articles/channels-devices-touchpoints/) is a list of inputs from left to right with simultaneous outputs below. Multi-modal maps include intents and gestures followed by voice and visual outputs. A complete map will provide a better sense of scope before building anything.
+
+Inspiration for this exercise comes from the [](https://www.thesprintbook.com/how)[Sketch Phase](https://www.thesprintbook.com/how) of _The Design Sprint_ and [How to Make an Experience Map](https://blog.carbonfive.com/2017/08/03/experience-mapping-2/).
```
```suggestion
Inspiration for this exercise comes from the [Sketch Phase](https://www.thesprintbook.com/how) of _The Design Sprint_ and [How to Make an Experience Map](https://blog.carbonfive.com/2017/08/03/experience-mapping-2/).
```

space-pope commented on content/docs/Design/map-out-integration.md:
```diff
@@ -0,0 +1,58 @@
+
+# **Getting Started**
+
+Setup will depend on whether you’ll be working with a **colocated** **team**, a **remote team**, or as an **individual**.
```
```suggestion
Setup will depend on whether you’ll be working with a **colocated team**, a **remote team**, or as an **individual**.
```

space-pope commented on content/docs/Design/map-out-integration.md:
```diff
@@ -0,0 +1,58 @@
+
+## **Remote teams**
+
+You need a shared document that everyone can access and a video conferencing tool. For shared documentation, we use [](https://www.google.com/sheets/about/)[Google Sheets](https://www.google.com/sheets/about/). For video conferencing, we use [](https://zoom.us/)[Zoom](https://zoom.us/).
```
Double link to both google and zoom here
```suggestion
You need a shared document that everyone can access and a video conferencing tool. For shared documentation, we use [Google Sheets](https://www.google.com/sheets/about/). For video conferencing, we use [Zoom](https://zoom.us/).
```

space-pope commented on content/docs/Design/map-out-integration.md:
```diff
@@ -0,0 +1,58 @@
+
+# **The exercise**
+
+## [**For existing apps**](/docs/Design/for-existing-apps) \*\*\*\*
```
weird trailing asterisks

space-pope commented on content/docs/Design/map-out-integration.md:
```diff
@@ -0,0 +1,58 @@
+
+We provide on-site and remote experience mapping workshops tailored to your organization.
+
+[Email us for more details →](mailto: hello@spokestack.io)
```
weird mailto rendering

space-pope commented on content/docs/Design/meet-your-team-face-to-face.md:
```diff
@@ -0,0 +1,29 @@
+
+## **Gauge familiarity with multi-modal**
+
+Dig a little bit deeper. Understand their relationship to voice-only as well multi-modal experiences. You might ask:
```
```suggestion
Dig a little bit deeper. Understand their relationship to voice-only as well as multi-modal experiences. You might ask:
```

space-pope commented on content/docs/Design/script-storyboard-responses.md:
```diff
@@ -0,0 +1,76 @@
+draft: false
+---
+
+When a user expresses an intent and your app responds, we call this interaction a **turn**. Your script will reflect turns you defined in your [experience map](/docs/Design/map-out-integration) in more detail. You’ll also want to reference your [initial list of intents](/docs/Design/map-out-integration).
```
Is that the right link for the second thing? Not sure which document we should be pointing at.

space-pope commented on content/docs/Design/script-storyboard-responses.md:
```diff
@@ -0,0 +1,76 @@
+
+# **4. Respond.**
+
+Focus on the conversational aspect of your response for now. We’ll get to [visual responses](/docs/Design/tips-for-designing-visual-output) in step #8. Refer to our [best practices](/docs/Design/tips-for-writing-dialog) if you get stuck. Label each turn with its corresponding intent. \*\*\*\*Write any variations below each response. Note any assumptions and remaining questions below each turn.
```
escaped asterisks

space-pope commented on content/docs/Design/script-storyboard-responses.md:
```diff
@@ -0,0 +1,76 @@
+# **8. Replace [square brackets] with visuals.**
+
+Now that you’ve established what you’re going to say, it’s time to make [UI decisions](/docs/Design/tips-for-designing-visual-output). We use [Sketch](https://www.sketch.com/) to wireframe and mock up ideas. Export screens from your tool of choice and pair them with corresponding turns.
+[Adobe XD](https://www.googleadservices.com/pagead/aclk?sa=L&ai=DChcSEwiXs56Lw8znAhUFpLMKHV0JCPgYABAAGgJxbg&ohost=www.google.com&cid=CAESQOD27NwYvUndVIkFoGVBtCUnQV--Xs17mozEEQUBNJhCu1KxlQBaUgjBLdBXTgG-MQK3vmhw-0w_igQAcqeKCaE&sig=AOD64_0Nv14O4kKFrl5i2FYPHzlG6DHP7w&q=&ved=2ahUKEwi8-5SLw8znAhWvSt8KHaACC2gQ0Qx6BAgTEAE&adurl=)
```
lot of overly verbose links in this section, and i think this line is actually a mistake
```suggestion
```

space-pope commented on content/docs/Design/script-storyboard-responses.md:
```diff
@@ -0,0 +1,76 @@
+
+Now that you’ve established what you’re going to say, it’s time to make [UI decisions](/docs/Design/tips-for-designing-visual-output). We use [Sketch](https://www.sketch.com/) to wireframe and mock up ideas. Export screens from your tool of choice and pair them with corresponding turns.
+[Adobe XD](https://www.googleadservices.com/pagead/aclk?sa=L&ai=DChcSEwiXs56Lw8znAhUFpLMKHV0JCPgYABAAGgJxbg&ohost=www.google.com&cid=CAESQOD27NwYvUndVIkFoGVBtCUnQV--Xs17mozEEQUBNJhCu1KxlQBaUgjBLdBXTgG-MQK3vmhw-0w_igQAcqeKCaE&sig=AOD64_0Nv14O4kKFrl5i2FYPHzlG6DHP7w&q=&ved=2ahUKEwi8-5SLw8znAhWvSt8KHaACC2gQ0Qx6BAgTEAE&adurl=)
+Create a prototype if it’s helpful. [Adobe XD](https://www.adobe.com/products/xd.html?sdid=12B9F15S&mv=Search&ef_id=CjwKCAiA4Y7yBRB8EiwADV1hacKOV9sHq556fq_QvGjvfCqqclV9lY0JL-nkJ-gTsUMp6ZA3UVYNThoC6mQQAvD_BwE:G:s&s_kwcid=AL!3085!3!315233774109!e!!g!!adobe%20xd) [AAdobe XDAAdobe XD](https://www.googleadservices.com/pagead/aclk?sa=L&ai=DChcSEwiXs56Lw8znAhUFpLMKHV0JCPgYABAAGgJxbg&ohost=www.google.com ...
```
looks like `Adobe XD` got duplicated
```suggestion
Create a prototype if it’s helpful. [Adobe XD](https://www.adobe.com/products/xd.html) is the best option we’ve found for prototyping with voice. Understand the [challenges](/docs/Design/test-with-real-people) of adding voice to a prototype before proceeding.
```

space-pope commented on content/docs/Design/starting-from-scratch.md:
```diff
@@ -0,0 +1,48 @@
+
+> **User.** This will include gestures and intents. Distinguish gestures from intents using quotes. For example, you could write "pace" on a sticky note or cell to capture a user who's asking how fast they’re running. Create separate maps for **return** and **new users**. Consider how you’ll onboard voice to new users. iOS has separate permissions for microphone & speech recognition. Android user ...
+>
+> **Voice.** This will include audio output. How will you respond to your user? When will you allow them to speak? Will they be able to interrupt? Prompts and responses should be in quotes. Don’t fill in this column if the feature/interaction in question isn’t relevant. This will serve as the basis for [your script](https://www.spoekstack.io/docs/Design/script-storyboard-responses) later on.
```
```suggestion
> **Voice.** This will include audio output. How will you respond to your user? When will you allow them to speak? Will they be able to interrupt? Prompts and responses should be in quotes. Don’t fill in this column if the feature/interaction in question isn’t relevant. This will serve as the basis for [your script](script-storyboard-responses) later on.
```

space-pope commented on content/docs/Design/starting-from-scratch.md:
```diff
@@ -0,0 +1,48 @@
+>
+> **Questions (optional).** Think of this map as a starting point. Make note of anything you’re unsure about.
+
+![As an example, we’ve started a map for an imagined fitness tracker, MyRunBuddy.](https://paper-attachments.dropbox.com/s_64309765F21EEABBFEB095F2281AAE0D852D3B39607DDCA807D5E9D53AAABE5C_1581629577075_MyRunBuddy+-+actors+new.png)
```
```suggestion
As an example, we’ve started a map for an imagined fitness tracker, MyRunBuddy. 
![](https://paper-attachments.dropbox.com/s_64309765F21EEABBFEB095F2281AAE0D852D3B39607DDCA807D5E9D53AAABE5C_1581629577075_MyRunBuddy+-+actors+new.png)
```

space-pope commented on content/docs/Design/starting-from-scratch.md:
```diff
@@ -0,0 +1,48 @@
+
+Map top to bottom, left to right until you capture your app as-is. Don’t worry about UI specifics at this point. Capture high-level inputs and outputs that get the user to the next step. Move actions around until the order represents your app’s flow. Look for any holes in the map that need further clarity. Document lapses in time where necessary.
+
+![It’s important to note when a new user would hear this greeting. For example, if the user is opening the app for the first time not using their voice, they might not expect to hear something.](https://paper-attachments.dropbox.com/s_64309765F21EEABBFEB095F2281AAE0D852D3B39607DDCA807D5E9D53AAABE5C_1581629849651_MyRunBuddy+-+new+column.png)
```
```suggestion
It’s important to note when a new user would hear this greeting. For example, if the user is opening the app for the first time not using their voice, they might not expect to hear something.
![](https://paper-attachments.dropbox.com/s_64309765F21EEABBFEB095F2281AAE0D852D3B39607DDCA807D5E9D53AAABE5C_1581629849651_MyRunBuddy+-+new+column.png)
```

space-pope commented on content/docs/Design/starting-from-scratch.md:
```diff
@@ -0,0 +1,48 @@
+
+Nothing is set in stone. That’s the beauty of using stickies or a spreadsheet. Move actions around until the order feels right to you. If intents follow the same pattern in your map, feel free to map one as an example for the rest. Once you’ve exhausted your initial list of intents, you’re done. Look for any holes in the map that need further clarity.
+
+![We mapped the user opening the app for the first time, creating an account, starting a run, and finishing a run. This does not include how a user would ask for their run stats (see “questions”). To account for this, you might either want to go back and try to fit these into your initial map or include these in your script later.](https://paper-attachments.dropbox.com/s_64309765F21EEABBFEB095F22 ...
```
```suggestion
We mapped the user opening the app for the first time, creating an account, starting a run, and finishing a run. This does not include how a user would ask for their run stats (see “questions”). To account for this, you might either want to go back and try to fit these into your initial map or include these in your script later.

![](https://paper-attachments.dropbox.com/s_64309765F21EEABBFEB095F2281AAE0D852D3B39607DDCA807D5E9D53AAABE5C_1581630226823_MyRunBuddy+-+new+complete.png)
```

space-pope commented on content/docs/Design/starting-from-scratch.md:
```diff
@@ -0,0 +1,48 @@
+![We mapped the user opening the app for the first time, creating an account, starting a run, and finishing a run. This does not include how a user would ask for their run stats (see “questions”). To account for this, you might either want to go back and try to fit these into your initial map or include these in your script later.](https://paper-attachments.dropbox.com/s_64309765F21EEABBFEB095F22 ...
+
+**5. Document your map (in-person teams only).**
+Transfer your map to a shared document that everyone can reference. We use [](https://www.google.com/sheets/about/)[Google Sheets](https://www.google.com/sheets/about/). Like with the stickies, include one intent or action per cell if using a spreadsheet. Make sure anyone on your team can interrupt your document without explanation.
```
```suggestion
Transfer your map to a shared document that everyone can reference. We use [Google Sheets](https://www.google.com/sheets/about/). Like with the stickies, include one intent or action per cell if using a spreadsheet. Make sure anyone on your team can interpret your document without explanation.
```

space-pope commented on content/docs/Design/starting-from-scratch.md:
```diff
@@ -0,0 +1,48 @@
+**5. Document your map (in-person teams only).**
+Transfer your map to a shared document that everyone can reference. We use [](https://www.google.com/sheets/about/)[Google Sheets](https://www.google.com/sheets/about/). Like with the stickies, include one intent or action per cell if using a spreadsheet. Make sure anyone on your team can interrupt your document without explanation.
+
+> For those using Google Sheets, here’s a [](https://docs.google.com/spreadsheets/d/1epKA1i_2Cbb8sCEnV_D1mHl4VHfdJhY7-EXZGIrPjbM/edit?usp=sharing)[template](https://docs.google.com/spreadsheets/d/1epKA1i_2Cbb8sCEnV_D1mHl4VHfdJhY7-EXZGIrPjbM/edit?usp=sharing) to get you started. Make sure to duplicate this document to make changes.
```
```suggestion
> For those using Google Sheets, here’s a [template](https://docs.google.com/spreadsheets/d/1epKA1i_2Cbb8sCEnV_D1mHl4VHfdJhY7-EXZGIrPjbM/edit?usp=sharing) to get you started. Make sure to duplicate this document to make changes.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+draft: false
+---
+
+Remember the survey and one-sheets you created while [getting to know your audience](/docs/Design/get-to-know-your-audience)? Refer to your findings from both when figuring out what and who to test your app with. What's your target audience’s comfort level with multi-modal interfaces and/or voice? We recommend testing a beta version of your app rather than a prototype.
```
```suggestion
Remember the survey and one-sheets you created while [getting to know your audience](/docs/Design/get-to-know-your-audience)? Refer to your findings from both when figuring out what and whom to test your app with. What's your target audience’s comfort level with multi-modal interfaces and/or voice? We recommend testing a beta version of your app rather than a prototype.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+
+# **Challenges of testing multi-modal prototypes**
+
+There are many script prototyping tools, but few that account for the addition of visuals. The three main tools we have tried include Sketch, Inision, Adobe XD.
```
```suggestion
There are many script prototyping tools, but few that account for the addition of visuals. The three main tools we have tried include Sketch, Invision, and Adobe XD.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+
+There are many script prototyping tools, but few that account for the addition of visuals. The three main tools we have tried include Sketch, Inision, Adobe XD.
+
+## Sketch & Invision
```
I'm not sure these headings make sense with no content.

roppem9 replied:
Yes, this article is not finished

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+
+## **Unmoderated (remote)**
+
+We recommend this if you're limited by time, man-power, and/or budget. Unmoderated candidates tend to give more honest feedback. One disadvantage is that you’re unable to help the user when they get stuck or ask follow-up questions. For remote testing, we use “task completion” tests for iOS, Android, or both on [UserBob](https://userbob.com/).
```
```suggestion
We recommend this if you're limited by time, manpower, and/or budget. Unmoderated candidates tend to give more honest feedback. One disadvantage is that you’re unable to help the user when they get stuck or ask follow-up questions. For remote testing, we use “task completion” tests for iOS, Android, or both on [UserBob](https://userbob.com/).
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+We recommend this if you're limited by time, man-power, and/or budget. Unmoderated candidates tend to give more honest feedback. One disadvantage is that you’re unable to help the user when they get stuck or ask follow-up questions. For remote testing, we use “task completion” tests for iOS, Android, or both on [UserBob](https://userbob.com/).
+
+1. **Write research questions.**
+   These should include uncertainties you want to put to rest. Don’t expose these to your users. Think of these as internal measurement tools. Here are some examples:
```
```suggestion
These should include uncertainties you want to put to rest. Don’t expose these to your users. Think of these as internal measurement tools. Here are some examples:
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+- Where applicable, do they understand that your app is wakeword-enabled? Did they use this feature? If not, why?
+
+2. **Provide a url for testing.**
+   This could include a link to the app or google play store. If testing a beta version of an app, send them an invitation before testing. Provide instructions not to download before the test.
```
```suggestion
This could include a link to the iTunes Store or Google Play Store. If testing a beta version of an app, send them an invitation before testing. Provide instructions not to download before the test.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+   This could include a link to the app or google play store. If testing a beta version of an app, send them an invitation before testing. Provide instructions not to download before the test.
+
+3. **Write a list of tasks based on your research questions, no more than 15.**
+   Include follow-up questions to tasks you want further clarity on. A good follow-up question to ask before a user completes a task is: What would you expect to happen here? Try to avoid questions that project future behavior. It’s hard to get honest feedback from projections.
```
```suggestion
Include follow-up questions to tasks you want further clarity on. A good follow-up question to ask before a user completes a task is: What would you expect to happen here? Try to avoid questions that project future behavior. It’s hard to get honest feedback from projections.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+   Include follow-up questions to tasks you want further clarity on. A good follow-up question to ask before a user completes a task is: What would you expect to happen here? Try to avoid questions that project future behavior. It’s hard to get honest feedback from projections.
+
+4. **Include a description of what the user should do.**
+   This of this as your intro. Add this before your list of tasks. Here’s an example:
```
```suggestion
Think of this as your intro. Add this before your list of tasks. Here’s an example:
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+4. **Include a description of what the user should do.**
+   This of this as your intro. Add this before your list of tasks. Here’s an example:
+
+> We’re asking you to test a mobile app that lets you record different types of fitness activities. This is a work in progress, so if something doesn’t work, explain what you would expect to happen and move on. We’re testing the app, not you - no need to worry about making mistakes.
```
```suggestion
> We’re asking you to test a mobile app that lets you record different types of fitness activities. This is a work in progress, so if something doesn’t work, explain what you would expect to happen and move on. We’re testing the app, not you — no need to worry about making mistakes.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+>
+> Below are a series of tasks. Try completing each task using the app on your phone. As much as possible, think out loud as you go along: say what you’re looking at, what you’re trying to do, and what you’re thinking.
+
+5. **Iterate on your tasks and description if users get hung up on phrasing or if there's a bug.** Remember, you’re collecting qualitative data, not quantitative. If you start to see users saying the same things, finish up that round of testing. Move on to tweaking your product before the next round of testing.
```
```suggestion
5. **Iterate on your tasks and description if users get hung up on phrasing or if there's a bug.** 

Remember, you’re collecting qualitative data, not quantitative. If you start to see users saying the same things, finish up that round of testing. Move on to tweaking your product before the next round of testing.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+
+## **Moderated**
+
+You’re able to ask more follow-up questions and improvise, but feedback can sometimes be less honest. We recommend this if you have more time and are able to meet your users face-to-face, either remote or co-located.
```
```suggestion
You’re able to ask more follow-up questions and improvise, but feedback can sometimes be less honest. We recommend this if you have more time and are able to meet your users face-to-face, either remote or colocated.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+
+You’re able to ask more follow-up questions and improvise, but feedback can sometimes be less honest. We recommend this if you have more time and are able to meet your users face-to-face, either remote or co-located.
+
+Use your survey to recruit candidates. For co-located tests, make sure candidates are local. Allow yourself a week or more to schedule time to meet one-on-one with candidates. We use Calendly and/or Google Calendar to schedule meetings and share testing times with teammates.
```
A shame we don't have links in this section
```suggestion
Use your survey to recruit candidates. For colocated tests, make sure candidates are local. Allow yourself a week or more to schedule time to meet one-on-one with candidates. We use Calendly and/or Google Calendar to schedule meetings and share testing times with teammates.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+
+Use your survey to recruit candidates. For co-located tests, make sure candidates are local. Allow yourself a week or more to schedule time to meet one-on-one with candidates. We use Calendly and/or Google Calendar to schedule meetings and share testing times with teammates.
+
+Emailing your candidates.
```
Should this be a section heading?

roppem9 replied:
Ah, this article isn't actually finished. I need to finish this first before it gets published...

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+Gather a list of tasks that you’re the most unsure about, no more than 10. What are you trying to learn from these candidates? Add these research questions to the top of your script. Write a test script. Here’s a sample to get you started.
+
+1. **Write research questions.**
+   These should include product uncertainties you want to put to rest. Don’t expose these to your users. Think of these as internal measurement tools. Here are some examples:
```
```suggestion
These should include product uncertainties you want to put to rest. Don’t expose these to your users. Think of these as internal measurement tools. Here are some examples:
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+- Where applicable, do they understand that your app is wakeword-enabled? Did they use this feature? If not, why?
+
+2. **Provide a url for testing.**
+   This could include a link to the app or google play store. If testing a beta version of an app, send them an invitation before testing. Provide instructions not to download before the test. It’s best of users to use their own device for testing so eliminate unforeseen barriers. Bring a few extra mobile devices for testing as a back-up in case users are unable to download your app.
```
```suggestion
This could include a link to the iTunes Store or Google Play Store. If testing a beta version of an app, send them an invitation before testing. Provide instructions not to download before the test. It’s best for users to use their own device for testing to eliminate unforeseen barriers. Bring a few extra mobile devices for testing as a back-up in case users are unable to download your app.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+   This could include a link to the app or google play store. If testing a beta version of an app, send them an invitation before testing. Provide instructions not to download before the test. It’s best of users to use their own device for testing so eliminate unforeseen barriers. Bring a few extra mobile devices for testing as a back-up in case users are unable to download your app.
+
+3. **Write a list of tasks based on your research questions, no more than 15.**
+   Include follow-up questions to tasks you want further clarity on. A good follow-up question to ask before a user completes a task is: What would you expect to happen here? Try to avoid questions that project future behavior. It’s hard to get honest feedback from projections.
```
```suggestion
Include follow-up questions to tasks you want further clarity on. A good follow-up question to ask before a user completes a task is: What would you expect to happen here? Try to avoid questions that project future behavior. It’s hard to get honest feedback from projections.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+   Include follow-up questions to tasks you want further clarity on. A good follow-up question to ask before a user completes a task is: What would you expect to happen here? Try to avoid questions that project future behavior. It’s hard to get honest feedback from projections.
+
+4. **Include a description of what the user should do.**
+   This of this as your intro. Add this before your list of tasks. Here’s an example:
```
```suggestion
This of this as your intro. Add this before your list of tasks. Here’s an example:
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+4. **Include a description of what the user should do.**
+   This of this as your intro. Add this before your list of tasks. Here’s an example:
+
+> We’re asking you to test a mobile app that lets you record different types of fitness activities. This is a work in progress, so if something doesn’t work, explain what you would expect to happen and move on. We’re testing the app, not you - no need to worry about making mistakes.
```
```suggestion
> We’re asking you to test a mobile app that lets you record different types of fitness activities. This is a work in progress, so if something doesn’t work, explain what you would expect to happen and move on. We’re testing the app, not you — no need to worry about making mistakes.
```

space-pope commented on content/docs/Design/test-with-real-people.md:
```diff
@@ -0,0 +1,85 @@
+>
+> Below are a series of tasks. Try completing each task using the app on your phone. As much as possible, think out loud as you go along: say what you’re looking at, what you’re trying to do, and what you’re thinking.
+
+5. **Iterate on your tasks and description if users get hung up on phrasing or if there's a bug.** Remember, you’re collecting qualitative data, not quantitative. If you start to see users saying the same things, finish up that round of testing. Move on to tweaking your product before the next round of testing.
```
```suggestion
5. **Iterate on your tasks and descriptions if users get hung up on phrasing or if there's a bug.** 

Remember, you’re collecting qualitative data, not quantitative. If you start to see users saying the same things, finish up that round of testing. Move on to tweaking your product before the next round of testing.
```

space-pope commented on content/docs/nav.json:
```diff
@@ -9,5 +9,17 @@
   "Speech Pipeline Configuration",
   "Wakeword Models",
-  "TTS"
+  "TTS",
```
I should change the navId of this page; good reminder.

space-pope commented on content/docs/iOS/getting-started.md:
```diff
@@ -31,10 +31,10 @@ in your terminal.
 ## Integration
 
-In order for your app to accept voice input via Spokestack, it needs three things:
+In order for your app to accept voice input via Spokestack, it needs four things:
```
Noel has updates to this page in his latest PR, but this is one I didn't catch. Since Noel self-merged, I think this change should win, so cool.

space-pope commented on content/docs/Design/tips-for-designing-visual-output.md:
```diff
@@ -0,0 +1,73 @@
+
+### Show that you’re listening & thinking
+
+This happens after a user asks a question, makes a command, or invokes a wakeword. How would a friend communicate that it’s your turn to talk? They might pause and maintain eye contact. These are visual and audible cues.
```
```suggestion
This happens after a user asks a question, gives a command, or invokes a wakeword. How would a friend communicate that it’s your turn to talk? They might pause and maintain eye contact. These are visual and audible cues.
```

space-pope commented on content/docs/Design/tips-for-designing-visual-output.md:
```diff
@@ -0,0 +1,73 @@
+
+This happens after a user asks a question, makes a command, or invokes a wakeword. How would a friend communicate that it’s your turn to talk? They might pause and maintain eye contact. These are visual and audible cues.
+
+The same goes for multi-modal experiences. Without a visual cue, a user could misinterpret silence as an error or lag in processing. Make your cue easy enough to spot from a distance. For example, Alexa is a headless device intended for hands-free, eyes-free communication. And yet, it makes use of an easy to spot visual cue: the blue ring.
```
```suggestion
The same goes for multi-modal experiences. Without a visual cue, a user could misinterpret silence as an error or lag in processing. Make your cue easy enough to spot from a distance. For example, Alexa is a headless device intended for hands-free, eyes-free communication. And yet, it makes use of an easy-to-spot visual cue: the blue ring.
```

space-pope commented on content/docs/Design/tips-for-designing-visual-output.md:
```diff
@@ -0,0 +1,73 @@
+
+Use an animation to show active listening. Static cues don’t convey activity in the same way. Create an animation that lasts the same duration as your user’s utterance. An animation that ends before the user's finished talking is a sign that their intent wasn't heard. Make sure your cues are always in the same place so users know where to look ([Golden Rule #1:](https://www.cs.umd.edu/users/ben/g ...
+
+### Be judicious with screen real-estate
```
```suggestion
### Be judicious with screen real estate
```

space-pope commented on content/docs/Design/tips-for-designing-visual-output.md:
```diff
@@ -0,0 +1,73 @@
+
+Decide whether visual turn-taking should occupy a full or [partial](https://uxdesign.cc/redesigning-siri-and-adding-multitasking-features-to-ios-70c2f1a1569b) screen. Using a full screen to communicate turn-taking isn't always realistic. Allow users to exit listening mode using gestures.
+
+If screen real estate permits, consider including visual prompts. Remind users what they can say. Think back to “[educate; don’t overburden](/docs/Design/tips-for-writing-dialog).”
```
```suggestion
If screen real estate permits, consider including visual prompts. Remind users what they can say. Think back to “[educate; don’t overburden](/docs/Design/tips-for-writing-dialog)”.
```

space-pope commented on content/docs/Design/tips-for-designing-visual-output.md:
```diff
@@ -0,0 +1,73 @@
+
+Take into account the context in which your user is using your app. If users are in a situation where they’re unable to view their screen, visuals might not be as useful.
+
+![Introducing visual cues for intents when a user is recording a run for the first time isn't useful.](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581541107663_MyRunBuddy+-+Start+a+run.png)
```
```suggestion
Introducing visual cues for intents when a user is recording a run for the first time isn't useful.

![](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581541107663_MyRunBuddy+-+Start+a+run.png)
```

space-pope commented on content/docs/Design/tips-for-designing-visual-output.md:
```diff
@@ -0,0 +1,73 @@
+
+Consider the type of information you’re confirming. Follow the same rules you would for [designing notifications](https://www.nngroup.com/articles/push-notification/). Provide enough information so that users know what's going on. Don’t distract the user. Confirmations need to be small, legible, and in a color palette that doesn’t interfere. Avoid colors that convey immediacy.
+
+![For this example, the app could either choose to a) implicitly end the run and save it or b) tell the user they have a run in progress and ask if they want to stop & save.](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581631804482_MyRunBuddy+-+Saving.png)
```
```suggestion
For this example, the app could either choose to a) implicitly end the run and save it or b) tell the user they have a run in progress and ask if they want to stop & save.

![](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581631804482_MyRunBuddy+-+Saving.png)
```

space-pope commented on content/docs/Design/tips-for-designing-visual-output.md:
```diff
@@ -0,0 +1,73 @@
+
+![For this example, the app could either choose to a) implicitly end the run and save it or b) tell the user they have a run in progress and ask if they want to stop & save.](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581631804482_MyRunBuddy+-+Saving.png)
+
+![The user then recorded a walk. Once saved, the user received visual confirmation.](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581543029296_MyRunBuddy+-+Record+walk.png)
```
```suggestion
The user then recorded a walk. Once saved, the user received visual confirmation.

![](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581543029296_MyRunBuddy+-+Record+walk.png)
```

space-pope commented on content/docs/Design/tips-for-designing-visual-output.md:
```diff
@@ -0,0 +1,73 @@
+
+If you need to use a follow-up, supplement with visuals ([Golden Rule #7:](https://www.cs.umd.edu/users/ben/goldenrules.html) “keep users in control”). If a user responds using gestures, providing audio output may not be appropriate.
+
+![In this example, the app asked a follow-up. The user responded by tapping a button and then continued by using gestures.](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581631877473_MyRunBuddy+-+Stop+a+run.png)
```
```suggestion
In this example, the app asked a follow-up. The user responded by tapping a button and then continued by using gestures.

![](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581631877473_MyRunBuddy+-+Stop+a+run.png)
```

space-pope commented on content/docs/Design/tips-for-designing-visual-output.md:
```diff
@@ -0,0 +1,73 @@
+
+If requested information isn’t already available, consider opening a different screen. Respect your user’s time. Don’t overcomplicate output for a complex query. Format information for quick retrieval, emphasizing utility over complexity. Display information that is relevant to the user’s intent.
+
+![In this example, the user requested information that wasn’t already on the screen. As a result, the screen reformatted information to match the user’s request.](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581631941998_MyRunBuddy+-+Distance.png)
```
```suggestion
In this example, the user requested information that wasn’t already on the screen. As a result, the screen reformatted information to match the user’s request.

![](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581631941998_MyRunBuddy+-+Distance.png)
```

space-pope commented on content/docs/Design/tips-for-designing-visual-output.md:
```diff
@@ -0,0 +1,73 @@
+
+Take into account users with accessibility needs. Make clear how they will receive output. Label visual inputs with corresponding actions: “see,” “hear,” “show me,” “tell me,” etc.
+
+![In this example, it’s clear what type of output the user will receive since they said ‘show me…’.](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581541264633_MyRunBuddy+-+Show+me.png)
```
```suggestion
In this example, it’s clear what type of output the user will receive since they said ‘show me…’.

![](https://paper-attachments.dropbox.com/s_BF5D22BAD9421AD3845A926151A97CA26F837572D6DA91B630753CEABD822986_1581541264633_MyRunBuddy+-+Show+me.png)
```

space-pope commented on content/docs/Design/tips-for-writing-dialogue.md:
```diff
@@ -0,0 +1,114 @@
+>
+> **MYRUNBUDDY** > [Create account] “Welcome to MyRunBuddy! Let’s start by creating an account.”
+
+_Since this user is new to MyRunBuddy, we’ve started by giving the user an instruction. This can_ _bee_ _brief when paired with a visual, in this case_ _a screen for account creation\*\*. Users aren’t going to want to input private account information using their voice._
```
```suggestion
_Since this user is new to MyRunBuddy, we’ve started by giving the user an instruction. This can be brief when paired with a visual, in this case a screen for account creation. Users aren’t going to want to input private account information using their voice._
```

space-pope commented on content/docs/Design/tips-for-writing-dialogue.md:
```diff
@@ -0,0 +1,114 @@
+>
+> **MYRUNBUDDY** > [Recording Run] “Starting. To check your run’s progress, say ‘What’s my pace?’ or ‘How far have I run?’”
+
+_The user heard these prompts because it's their first time recording a run. Use your best judgement with what types of intents to include here. Revealing how to_ _phrase_ _one intent might clue them in on how to say other similar intents. This is something worth_ [_testing_](/dos/design/test-with-real-people/user-testing) _with your users._
```
```suggestion
_The user heard these prompts because it's their first time recording a run. Use your best judgement with what types of intents to include here. Revealing how to phrase one intent might clue them in on how to say other similar intents. This is something worth [testing](/dos/design/test-with-real-people/user-testing) with your users._
```

space-pope commented on content/docs/Design/tips-for-writing-dialogue.md:
```diff
@@ -0,0 +1,114 @@
+>
+> **MYRUNBUDDY** > [Confirmation + Recording Run] “Starting…”
+
+_Since this user has recored a run before using their voice, there’s no follow-up to the app’s confirmation. We’ll_ [_show you examples_](/docs/Design/tips-for-designing-visual-output) _of things you might want to include along with a confirmation in later steps._
```
```suggestion
_Since this user has recored a run before using their voice, there’s no follow-up to the app’s confirmation. We’ll [show you examples](/docs/Design/tips-for-designing-visual-output) of things you might want to include along with a confirmation in later steps._
```

space-pope commented on content/docs/Design/tips-for-writing-dialogue.md:
```diff
@@ -0,0 +1,114 @@
+
+_Since this user has recored a run before using their voice, there’s no follow-up to the app’s confirmation. We’ll_ [_show you examples_](/docs/Design/tips-for-designing-visual-output) _of things you might want to include along with a confirmation in later steps._
+
+Confirmations don’t always need to be conversational. Consider sounds. In the example above, instead of saying “starting,” you could replace this with a beep or swoosh soup.
```
```suggestion
Confirmations don’t always need to be conversational. Consider sounds. In the example above, instead of saying “starting,” you could replace this with a beep or swoosh sound.
```

space-pope commented on content/docs/Design/tips-for-writing-dialogue.md:
```diff
@@ -0,0 +1,114 @@
+
+## 3. Be relevant
+
+Voice input for complex queries is faster and more convenient. Especially for users who are multi-tasking. And yet, the average person can read faster than they can listen. Consider whether a voice response, a [visual response](/docs/Design/tips-for-designing-visual-output), or both is appropriate. Respect your users’ time; don’t inconvenience them by taking too long to respond. A well-crafted re ...
```
```suggestion
Voice input for complex queries is faster and more convenient, especially for users who are multi-tasking. And yet, the average person can read faster than they can listen. Consider whether a voice response, a [visual response](/docs/Design/tips-for-designing-visual-output), or both is appropriate. Respect your users’ time; don’t inconvenience them by taking too long to respond. A well-crafted response ([Golden Rule #3:](https://www.cs.umd.edu/users/ben/goldenrules.html) “offer informative feedback”) will increase re-engagement.
```

space-pope commented on content/docs/Design/tips-for-writing-dialogue.md:
```diff
@@ -0,0 +1,114 @@
+>
+> **MYRUNBUDDY** > [Recording Run] “Stopping…Would you like to save this run?”
+
+_In this example, the app misheard the user’s_ _‘stop’ intent for_ _‘start’. This goes back to step #2: “confirm information”. The app let the user confirm an important action (in this case, ending a workout) before taking it. This prevents costly errors before they happen._
```
```suggestion
_In this example, the app misheard the user’s ‘stop’ intent for ‘start’. This goes back to step #2: “confirm information”. The app let the user confirm an important action (in this case, ending a workout) before taking it. This prevents costly errors before they happen._
```

space-pope commented on content/docs/Design/tips-for-writing-dialogue.md:
```diff
@@ -0,0 +1,114 @@
+
+_In this example, the app added a question to make the app sound more human. When the user answered the question, the app failed to respond as this question wasn’t supported by a yes/no intent._
+
+Include scenarios in your script for your app failing to receive speech input at all. What if they take longer than anticipated to answer a question? Account for scenarios in which users aren't sure how to phrase their question or response. What if there’s a long pause? You’ll want to re-prompt the user if this occurs. For example, if Siri doesn’t hear anything after a while, you might hear “Uh h ...
```
```suggestion
Include scenarios in your script for your app failing to receive speech input at all. What if they take longer than anticipated to answer a question? Account for scenarios in which users aren't sure how to phrase their question or response. What if there’s a long pause? You’ll want to re-prompt the user if this occurs. For example, if Siri doesn’t hear anything after awhile, you might hear “Uh huh,” “I’m here,” or “Yes?”
```

space-pope commented on content/docs/Design/tips-for-writing-dialogue.md:
```diff
@@ -0,0 +1,114 @@
+>
+> **MYRUNBUDDY** > [Recording Run + Map] "You’re. 5 miles away from the 3 mile marker.”
+
+_In this example, the response would read as_ _“you’re [pause] five miles away…” - a big difference!_ A good TTS engine can handle decimals with correct period placement. _To make this more conversational, change this to something like_ _“You’re half a mile away from the 3 mile marker.”_
```
```suggestion
_In this example, the response would read as “you’re [pause] five miles away…” - a big difference!  A good TTS engine can handle decimals with correct period placement. To make this more conversational, change this to something like, “You’re half a mile away from the 3 mile marker.”
```

space-pope commented on content/docs/Design/tips-for-writing-dialogue.md:
```diff
@@ -0,0 +1,114 @@
+
+Need help executing? We can provide the linguistic support you need for your next project.
+
+[Email us for more details →](mailto: hello@spokestack.io)
```
weird mailto link again

roppem9 replied:
yea, it's been a while since I've formatted things in html. Wasn't sure if that was formatted correctly or not?

timmywil replied:
It looked okay locally.

space-pope commented on content/docs/Design/map-out-integration.md:
```diff
@@ -0,0 +1,58 @@
+
+## **Individuals**
+
+You need a shared document in case you need to gather feedback and share your map later on. We use [](https://www.google.com/sheets/about/)[Google Sheets](https://www.google.com/sheets/about/).
```
```suggestion
You need a shared document in case you need to gather feedback and share your map later on. We use [Google Sheets](https://www.google.com/sheets/about/).
```

space-pope reviewed (commented):
OK, hope I caught all the little things here. I have a couple questions that maybe only @roppem9 can answer.

timmywil committed:
fixup! apply code suggestions from josh

Co-Authored-By: Josh <space-pope@users.noreply.github.com>

timmywil committed:
Merge branch 'develop' into feature/design

roppem9 committed:
Fixed weird asterisks on line 18

roppem9 committed:
Fixed wording on lines 15-16

roppem9 committed:
Fixed asterisks on line 46

roppem9 committed:
Removed 2nd link in line 8

Probably don't need to link to the same article twice in one paragraph, even though they are both referencing the same doc

roppem9 committed:
Removed excess asterisks in line 39

timmywil committed:
Merge branch 'develop' into feature/design

timmywil committed:
docs(design): format using prettier

timmywil commented:
@roppem9 If it's not too much trouble, I think the only remaining item is for you to finish the "Test with real people" article here on github.

timmywil commented:
@roppem9 I'm going to merge this before it gets too stale. Send me the latest for "Test with real people" whenever you're ready.

timmywil committed:
Merge branch 'develop' into feature/design

timmywil commented:
:tada: This issue has been resolved in version 1.7.1 :tada:

The release is available on [GitHub release](https://github.com/spokestack/spokestack-website/releases/tag/1.7.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
